marimo/_islands/island_generator.py

```diff
@@ -153,7 +153,7 @@ class MarimoIslandGenerator:
     # Example
 
     ```python
-    from marimo.islands import MarimoIslandGenerator
+    from marimo._islands import MarimoIslandGenerator
```

Comment:
is this `from marimo.islands import MarimoIslandGenerator` broken? ideally they dont need to import private paths